src/utils/atomWithObservable.ts

```diff
@@ -29,69 +29,52 @@ type ObservableLike<T> = {
 
 type SubjectLike<T> = ObservableLike<T> & Observer<T>
 
+type InitialDataFunction<T> = () => T | undefined
+
+export type AtomWithObservableOptions<TData> = {
+  initialData?: TData | InitialDataFunction<TData>
+}
+export type CreateObservableOptions<TData> = {
+  initialData?: TData
```

Comment:
Can we name it `initialValue`?
```suggestion
  initialValue?: TData
```

Reply:
Renamed everywhere to initialValue